ubuntu/derived/standard/README.md

```diff
@@ -4,7 +4,7 @@ The Ubuntu-based standard images include the following tools:
 - Basic command-line utilities (curl, ftp, etc.)
 - Essential build tools (gcc, make, etc.)
 - Python and Python 3
-- OpenJDK 7 (Ubuntu 14.04) or 8 (Ubuntu 16.04)
+- Oracle JDK 6, 7 (Default JDK on 14.04) and 8 (Default JDK on 16.04)
```

Comment:
This information is also in the main repo's README.md file - can you update it there as well?